advanced/editor-control-identifiers.md

```diff
@@ -11,187 +11,187 @@ Use the [toolbar]({{ site.baseurl }}/configure/editor-appearance/#toolbar) or th
+| pagebreak              | [pagebreak]({{ site.baseurl }}/plugins/pagebreak/)                     | Insert                                                                                                   | Inserts a pagebreak into the editor.                                                                      |
+| pageembed              | [pageembed]({{ site.baseurl }}/plugins/pageembed/)                     | Insert                                                                                                   | Opens the insert or edit iframe dialog.                                                                   |
+| pastetext              | [paste]({{ site.baseurl }}/plugins/paste/)                             | Edit                                                                                                     | Toggles paste as plain text on/off.                                                                       |
+| configurepermanentpen  | [permanentpen]({{ site.baseurl }}/plugins/permanentpen/)               | _Not Applicable_                                                                                         | Opens the Permanent Pen properties dialog.                                                                |
+| permanentpen           | [permanentpen]({{ site.baseurl }}/plugins/permanentpen/)               | _Not Applicable_                                                                                         | Enables and disables Permanent Pen formatting.                                                            |
+| preview                | [preview]({{ site.baseurl }}/plugins/preview/)                         | File                                                                                                     | Previews the current document.                                                                            |
+| print                  | [print]({{ site.baseurl }}/plugins/print/)                             | File                                                                                                     | Prints the current document.                                                                              |
+| searchreplace          | [searchreplace]({{ site.baseurl }}/plugins/searchreplace/)             | Edit                                                                                                     | Opens the search/replace dialog.                                                                          |
+| spellchecker           | [spellchecker]({{ site.baseurl }}/plugins/spellchecker/)               | Tools                                                                                                    | Toggles the spellchecker on/off.                                                                          |
```

Comment:
Not sure if you're worried, but this actually applies to both `spellchecker` and `tinymcespellchecker`.

Reply:
damn.  thanks, thought it might. I'll add a link to the tinymcespellchecker docs too. :)

Reply:
Updated